semantic-web-technology-introduction.markdown

```diff
@@ -57,27 +56,27 @@ Proof provides trace or explanation of the steps of logical reasoning and needs
 
 ### Trust
 A trust layer offers authentication of identity and evidence of the trustworthiness of data, services, and agents.  
-All but the last layer subjects are further discussed in separate sections.  
+All but the last layer subjects are further discussed in separate sections.# 
```

Comment:
Link?

Reply:
I'll add this one:
http://wifo5-03.informatik.uni-mannheim.de/bizer/pub/CarrollBizer-Trust-WWW2004-DevDay.pdf